packit_service/service/events.py

```diff
@@ -1061,6 +1123,28 @@ def from_build_id(
             topic, build_id, build, chroot, status, owner, project_name, pkg, timestamp
         )
 
+    @classmethod
+    def from_event_dict(cls, event: dict):
+        topic = event.get("topic")
+        project_name = event.get("project_name")
+        owner = event.get("owner")
+        build_id = event.get("build_id")
+        chroot = event.get("chroot")
+        timestamp = event.get("timestamp")
+        pkg = event.get("pkg")
+        status = event.get("status")
+
+        return CoprBuildEvent.from_build_id(
+            topic=topic,
+            build_id=build_id,
+            chroot=chroot,
+            status=status,
+            owner=owner,
+            project_name=project_name,
+            pkg=pkg,
+            timestamp=timestamp,
```

Comment:
Same question as above. Can we get rid of those one-time variables and, ideally, key hardcoding?